.github/workflows/make_bundle_conda.yml

```diff
@@ -76,7 +76,9 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-          repository: conda-forge/napari-feedstock
+          # repository: conda-forge/napari-feedstock
+          repository: jaimergp/napari-feedstock
+          ref: change-outputs
```

Comment:
This is temporary until https://github.com/conda-forge/napari-feedstock/pull/48 is merged